mne_qt_browser/_pg_figure.py

```diff
@@ -4318,11 +4323,12 @@ def _toggle_scalebars(self):
 
     def _overview_mode_changed(self, new_mode):
         self.mne.overview_mode = new_mode
+        self._save_setting("overview_mode", new_mode)
         self.mne.overview_bar.set_background()
         if not self.mne.overview_bar.isVisible():
             self._toggle_overview_bar()
 
-    def _overview_radio_clicked(self, checked, *, menu, new_mode):
+    def _overview_radio_clicked(self, checked=False, *, menu, new_mode):
```

Comment:
Not surprised this new default is required, I had to do something somewhere else fairly recently for 6.9+ PySide